pkg/controller/unifiedpushserver/unifiedpushserver_controller.go

```diff
@@ -939,6 +978,50 @@ func (r *ReconcileUnifiedPushServer) Reconcile(request reconcile.Request) (recon
 	}
 	//#endregion
 
+	//#region Monitoring
+	//## region ServiceMonitor
+	serviceMonitor := &monitoringv1.ServiceMonitor{ObjectMeta: metav1.ObjectMeta{Name: "unifiedpush", Namespace: instance.Namespace}}
+	op, err := controllerutil.CreateOrUpdate(context.TODO(), r.client, serviceMonitor, func(ignore runtime.Object) error {
+		reconcileServiceMonitor(serviceMonitor)
+		return nil
```

Comment:
```suggestion
		err := controllerutil.SetControllerReference(instance, serviceMonitor, r.scheme)
		return err
```

^Similar to in the main.go, all of these resources should have owner references set on them